### DISCUSSION TITLE
"Interactive" installer for Raspbian

### DISCUSSION THREAD

**Discussion by SigmaTel71** [debian-pi/raspbian-ua-netinst#533] [General]
It's a good idea to have unattended netinstaller for Raspbian, but configuring things through .txt file is a little complicated for me. Is there an "interactive" netinstaller for Raspbian to install operating system the way like Debian netinstall CD offers to do so?

**Comment by diederikdehaas** [comment]
> Is there an "interactive" netinstaller for Raspbian

Not that I'm aware of.

> configuring things through .txt file is a little complicated

What do you find complicated about it?
It is not (yet?) required to provide an `installer-config.txt` file, that is only needed to customize the installation. If not provided, it will use the default values of the configuration settings.
